facebook/facebook.c

```diff
@@ -142,14 +142,17 @@ fb_cb_api_connect(FbApi *api, gpointer data)
     account_t *acct;
     FbData *fata = data;
     struct im_connection *ic;
+    gint show_unread;
 
     ic = fb_data_get_connection(fata);
     acct = ic->acc;
 
+    show_unread = !(ic->flags & OPT_LOGGED_IN);
```

Comment:
I agree the name isn't the best, but it is needed. `icmb_connected` sets the flag `OPT_LOGGED_IN` in `ic->flags` if it's not already set. So I need to store if this flag was set before `icmb_connected` is called.

I'll change the name of the variable to something more describing.